DP21-JP/c100411014.lua

```diff
@@ -50,4 +50,19 @@ function s.activate(e,tp,eg,ep,ev,re,r,rp)
 			Duel.MoveSequence(mg:GetFirst(),1)
 		end
 	end
+	local c=e:GetHandler()
+	if e:IsHasType(EFFECT_TYPE_ACTIVATE) then
+		local e1=Effect.CreateEffect(c)
+		e1:SetType(EFFECT_TYPE_FIELD)
+		e1:SetCode(EFFECT_CANNOT_SPECIAL_SUMMON)
+		e1:SetProperty(EFFECT_FLAG_PLAYER_TARGET)
+		e1:SetTargetRange(1,0)
+		e1:SetTarget(c61884774.splimit)
+		e1:SetReset(RESET_PHASE+PHASE_END)
+		Duel.RegisterEffect(e1,tp)
+	end
+end
+	--Restricted to ritual monsters for rest of the turn
+function s.splimit(e,c)
+	return not c:IsRitualMonster
```

Comment:
This is my fault for not being clear when I mentioned the function before, but this should be IsRitualMonster(), with the brackets